test/e2e/metrics_test.go

```diff
@@ -0,0 +1,43 @@
+package e2e
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	core "k8s.io/api/core/v1"
+)
+
+// testMetrics deploys a Linux pod and tests that it can communicate with the Windows node's metrics port
+func testMetrics(t *testing.T) {
+	testCtx, err := NewTestContext(t)
+	require.NoError(t, err)
+
+	// Need at least one Windows node to run these tests, throwing error if this condition is not met
+	require.GreaterOrEqualf(t, len(gc.nodes), 1, "insufficient number of Windows nodes to run tests across"+
+		" nodes, Minimum node count: 1, Current node count: %d", len(gc.nodes))
```

Comment:
Lets be consistent about this check. Moreover it is a bit wordy:
`require.Greater(t, len(gc.nodes), 0, "test requires at least one Windows node to run")